sw/host/opentitanlib/src/spiflash/flash.rs

```diff
@@ -76,7 +76,7 @@ impl SpiFlash {
     pub const READ_SFDP: u8 = 0x5a;
     pub const NOP: u8 = 0x00;
     pub const RESET_ENABLE: u8 = 0x66;
-    pub const RESET: u8 = 0x99;
+    pub const RESET: u8 = 0xf0;
```

Comment:
I think the sequence RSTEN, RST (0x66, 0x99) is somewhat standard across flash parts.

- [MX66L2G45G](https://www.mxic.com.tw/Lists/Datasheet/Attachments/8697/MX66L2G45G,%203V,%202Gb,%20v1.0.pdf) Section 10-40, page 39.
- [W25Q128JV](https://www.winbond.com/resource-files/W25Q128JV%20RevI%2008232021%20Plus.pdf) Section 6.4, page 11.

If we're going to use a non-standard opcode and specify it in the SFDP, we should use the SFDP in the `chip_reset` function.

Reply:
The reason why we chose `0xf0` was that we wouldn't be able to enforce the 66/99 sequence anyway. 

Soft reset and rescue sequence support is defined in JESD216F.01, page 41 (bits 13:8 of the 16th DWORD).

We can also support both sequences or just the 66/99 if you prefer.

Reply:
Updated our RESET opcode to 0x99, ptal.